src/modules/Dropdown/Dropdown.js

```diff
@@ -1047,8 +1047,13 @@ export default class Dropdown extends Component {
     } else if (hasValue) {
       _text = _.get(this.getItemByValue(value), 'text')
     }
-
-    return <div className={classes}>{_text}</div>
+    let finalRenderedText = null
+    if (classes === 'text') {
```

Comment:
I update the code. Now returns always the rendered text with the aria-labels suggested. I tested the documentation page, and all dropdown added work as in the main documentation page (http://react.semantic-ui.com)